tests/README.md

```diff
@@ -1,5 +1,5 @@
-1. Before testing install ChromeDriver from:
-    http://chromedriver.chromium.org/
-2. Add ChromeDriver to PATH.
-3. go to directory *\melvil
-4. run command 'pytest'
\ No newline at end of file
+How to run tests:
+
+1. insert into .env email address and password
```

Comment:
This will not work for our CI testing enviroment. We don't use env in Travis. Probably we should rebuild test code for mail sending, use some test mail.server istead of gmail and add that info to TestConfig.